dev/com.ibm.ws.session.cache/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -14,3 +14,15 @@
 #NLS_ENCODING=UNICODE
 #NLS_MESSAGEFORMAT_NONE
 #
+
+httpSessionCache=HTTP Session Cache
+httpSessionCache.desc=Controls how HTTP sessions are persisted using JCache.
```

Comment:
I would suggest removing all periods at the end of the descriptions as they are not needed

Reply:
Is that a new policy?  I was always told that descriptions should end in a period and a quick look at several files around the product all showed descriptions ending with periods

Reply:
@rw2513